styles.css

```diff
@@ -0,0 +1,266 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  scroll-behavior: smooth;
+  font-family: 'Inter', sans-serif;
+}
+
+h2 {
+  font-size: 2.5rem;
+  color: #172b4d;
+}
+
+p {
+  color: #172b4d;
+  font-size: 1.25rem;
+}
+
+hr {
+  border: 4px solid #36b37e;
+  width: 50px;
+  margin-top: 24px;
+  margin-bottom: 60px;
+}
+
+.black {
+  margin: 0 auto;
+  width: 120px;
+  border-color: black;
+  margin-bottom: 10px;
+  margin-top: 300px;
+  border-radius: 4px;
+}
+
+i {
+  font-size: 1rem;
+}
+
+.menu {
+  margin: 24px;
+  color: black;
+}
+
+.header {
+  height: 50px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.welcome {
+  font-size: 1.25rem;
+  font-weight: 600;
+  margin: 24px;
+  color: #28352f;
+}
+
+.nav ul {
+  display: none;
+}
+
+main {
+  background: white url("Images/header-illsutration-mobile@2x.png") 0 0 no-repeat;
+  background-size: contain;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.headline .item {
+  margin: 60px 24px 0;
+  text-align: center;
+}
+
+.green {
+  color: #36b37e;
+}
+
+.socials {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.socials i {
+  margin: 20px;
+}
+
+.arrow {
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  outline: none;
+  border: 1px solid #dfe1e6;
+  background-color: white;
+  margin-left: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  margin-bottom: 68px;
+}
```

Comment:
- [ ] According to the requirement the main container should be displayed: grid; please implement so.

Reply:
completed